### PR TITLE
Update Chapter 3 & 6 slide notes to align subchapter numbering to textbook.

### PR DESCRIPTION
**Pull request by rserran**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -13,20 +13,28 @@ Imports:
...
     modeldata, 
     moments, 
+    parttree, 
     pdp,
     recipes,
-    rmarkdown,
+    rmarkdown, 
+    rpart, 
+    rpart.plot, 
+    rattle
```
Comment (jonthegeek) [inline DESCRIPTION]:
```suggestion
    rattle,
```

**Review by jonthegeek — CHANGES_REQUESTED** [review]
Need to add a missing comma to DESCRIPTION, otherwise it looks good as long as it builds!

**Review by jonthegeek — DISMISSED** [review]
Looks good as long as it builds!

**Review by jonthegeek — DISMISSED** [review]
Hopefully this will do it!

**Comment by rserran** [comment]
I jumped the gun and commit a fix to DESCRIPTION while the repo was updating.

**Review by jonthegeek — APPROVED** [review]
Hopefully adding plotly will be the last of it! I'm building locally but I figured I should let the workflow run in case it's done.

**Pull request merged**